newdle/client/src/actions.js

```diff
@@ -148,6 +149,10 @@ export function setPrivate(isPrivate) {
   return {type: SET_PRIVATE, private: isPrivate};
 }
 
+export function setNotification(isNotified) {
+  return {type: SET_NOTIFICATION, notify: isNotified};
+}
```

Comment:
```suggestion
export function setNotification(notify) {
  return {type: SET_NOTIFICATION, notify};
}
```

In `setPrivate` we only need the different name because using `private` there caused some error (since `private` is considered a reserved word in some contexts).